docs/assembly.rst

```diff
@@ -947,11 +989,14 @@ Pseudocode::
       POP
     LabelDefinition(name [id1, ..., idn] :) ->
       JUMPDEST
-      // register new variables id1, ..., idn and set the stack height to
-      // stack_height_at_block_start + number_of_local_variables
+      // register new local variables id1, ..., idn and adjust the stack
+      // height such that it matches the stack height at the beginning of
+      // the block plus all local variables including the just registered
+      // ones where idn is at the stack top. If n is zero, resets the stack
+      // height to just the local variables.
```

Comment:
Why isn't it possible to put new variables at the top of the stack, on top of all additional stack height adjustments.